daemon/start.go

```diff
@@ -33,11 +33,11 @@ func serviceSpec() (spec container.ServiceOptions, err error) {
 	}
 	spec = container.ServiceOptions{
 		Namespace: Namespace(),
-		Image:     viper.GetString(config.DaemonImage),
+		Image:     viper.GetString(config.CoreImage),
 		Env: []string{
 			"MESG.PATH=/mesg",
-			"API.SERVICE.SOCKETPATH=" + filepath.Join(viper.GetString(config.MESGPath), "server.sock"),
-			"SERVICE.PATH.HOST=" + filepath.Join(viper.GetString(config.MESGPath), "services"),
+			"API.SERVICE.SOCKETPATH=" + filepath.Join(viper.GetString(config.MESGPath), "server.sock"), // TODO: this one seems useless because MESGPath is already set.
```

Comment:
not useless because here the path will be from the `MESGPath` while executing this command so from the CLI so with the `~/.mesg` of the user, if not set, inside the docker it will bind to the `MESG.PATH` that is `/mesg` so the link will not be good because on the host user will not have `/mesg`